File: FizzBuzz.lua
function fizzBuzz(n)
    for i = 1, n do
        if i % 3 == 0 and i % 5 == 0 then
            print("FizzBuzz")
        elseif i % 3 == 0 then
            print("Fizz")
        elseif i % 5 == 0 then
            print("Buzz")
        else
            print(i)
        end
    end
end

-- Main program
io.write("Enter the value of n: ")
local n = tonumber(io.read())

-- Call the fizzBuzz function with the given n
fizzBuzz(n)

File: FizzBuzz.r
fizzbuzz <- function(n) {
  for (i in 1:n) {
    if (i %% 3 == 0 && i %% 5 == 0) {
      cat("FizzBuzz ", sep = "")
    } else if (i %% 3 == 0) {
      cat("Fizz ", sep = "")
    } else if (i %% 5 == 0) {
      cat("Buzz ", sep = "")
    } else {
      cat(i, " ", sep = "")
    }
  }
}

# Get input for n
n <- as.integer(readline("Enter the value of n: "))

# Call the fizzbuzz function with the given n
fizzbuzz(n)
